File: app/src/main/java/com/ikmr/banbara23/yaeyama_liner_checker/model/weather/Temperature.java
package com.ikmr.banbara23.yaeyama_liner_checker.model.weather;

/**
 * 気温
 */
public class Temperature {

    private String hight;
    private String low;

    public String getHight() {
        return this.hight;
    }

    public void setHight(String hight) {
        this.hight = hight;
    }

    public String getLow() {
        return this.low;
    }

    public void setLow(String low) {
        this.low = low;
    }

    @Override
    public String toString() {
        return "Temperature{" +
               "hight='" + hight + '\'' +
               ", low='" + low + '\'' +
               '}';
    }
}

File: domain/src/main/kotlin/com/yaeyama/linerchecker/domain/weather/Temperature.kt
package com.yaeyama.linerchecker.domain.weather

/**
 * 気温
 */
data class Temperature(
    /** 最高気温（Firebase のキーに合わせて綴りは hight のまま） */
    val hight: String = "",
    /** 最低気温 */
    val low: String = "",
)
